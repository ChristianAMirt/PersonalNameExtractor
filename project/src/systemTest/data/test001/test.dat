<NER>by <PER>A J Sanderson</PER> * and <PER>E Thivillier</PER> *</NER>
<NER>by <PER>Amy Tank</PER>, <PER>Denis Rickman</PER>, <PER>Jay Ehrgott</PER>, <PER>Richard Pearson</PER>, and 
 <PER>Jason Angel</PER></NER>
<NER>by <PER>Amy Tank</PER>, <PER>Denis Rickman</PER>, <PER>Jay Ehrgott</PER>, <PER>Richard Pearson</PER>,</NER>
<NER>by <PER>Andrew A. Thompson</PER></NER>
<NER>by <PER>CAPT Stephanie Kelley</PER> 
 Strategic Insights is a monthly electronic journal produced by the Center for Contemporary 
 Conflict at the Naval Postgraduate School in Monterey, California. The views expressed here are 
 those of the author (s) and do not necessarily represent the views of NPS, the Department of 
 Defense, or the U. S. Government. 
 For a PDF version of this article, click here</NER>
<NER>by <PER>CAPT Stephanie Kelley</PER></NER>
<NER>by <PER>CAPT. Nichole M. Harris</PER>, USAF 
 Strategic Insights is a monthly electronic journal produced by the Center for Contemporary 
 Conflict at the Naval Postgraduate School in Monterey, California. The views expressed here are 
 those of the author (s) and do not necessarily represent the views of NPS, the Department of 
 Defense, or the U. S. Government. 
 For a PDF version of this article, click here</NER>
<NER>by <PER>CAPT. Nichole M. Harris</PER>,</NER>
<NER>by <PER>Charles Rong</PER>, <PER>Deryn Chu</PER>, and <PER>John Hopkins</PER></NER>
<NER>by <PER>Christos E. Maragoudakis</PER> and <PER>Vernon Kopsa</PER></NER>
<NER>by <PER>Clifford Yapp</PER></NER>
<NER>by <PER>Daniel A. Pinkston</PER></NER>
<NER>by <PER>David Y. T. Chiu</PER> and <PER>Troy Alexander</PER></NER>
<NER>by <PER>Douglas J. Strand</PER></NER>
<NER>by <PER>Dr. John K. Brennan, Dr.</PER> <PER>Linhbao Tran</PER>, 
 and Dr. M ? ge</NER>
<NER>by <PER>Dr. John K. Brennan, Dr.</PER> <PER>Linhbao Tran</PER>,</NER>
<NER>by <PER>E. J. Rapacki, Jr.</PER></NER>
<NER>by <PER>Faye R. Toulan</PER>, <PER>Christopher Stabler</PER>, <PER>John J. La Scala</PER>, <PER>Hank Feuer</PER>, 
 <PER>Dave Flanagan</PER>, and <PER>Paul Touchet</PER></NER>
<NER>by <PER>Faye R. Toulan</PER>, <PER>Christopher Stabler</PER>, <PER>John J. La Scala</PER>, <PER>Hank Feuer</PER>,</NER>
<NER>by <PER>G. B. Vunni</PER></NER>
<NER>by <PER>Getachew Kirose</PER></NER>
<NER>by <PER>Govind Mallick</PER> and <PER>Shashi P. Karna</PER></NER>
<NER>by <PER>Hiralal Khatri</PER>, <PER>Kenneth Ranney</PER>, 
 <PER>Kwok Tom</PER>, and Romeo del</NER>
<NER>by <PER>Hiralal Khatri</PER>, <PER>Kenneth Ranney</PER>,</NER>
<NER>by <PER>Ian M. Standley</PER>, and <PER>W. Thomas Pike</PER>, Kinemetrics, Inc., 22 2 Vista Avenue</NER>
<NER>by <PER>James D. Walrath</PER></NER>
<NER>by <PER>James DeSpirito</PER> and <PER>Mary S. Binseel</PER></NER>
<NER>by <PER>Jean ISLER</PER> (2) - <PER>Jean G. GOLIGER</PER> (3) - <PER>Daniel BOCHAND</PER> (4) 
 <PER>Georges QUEROL</PER> (5) - <PER>Louis PICARD</PER> (6) and <PER>Jo ? l FERRON</PER> (7) 
 CLUB MURAT - BP 12 9 
 Tel : (33) - (1) 39.46.15.50 
 Fax : (33) - (1) 39.46.15.3</NER>
<NER>by <PER>Jessie Y. C. Chen</PER> and <PER>Peter I. Terrence</PER></NER>
<NER>by <PER>John J. La Scala</PER>, <PER>Scott Bingham</PER>, <PER>Kevin S. Andrews</PER>, <PER>James M. Sands</PER>, and 
 <PER>Giuseppe R. Palmese</PER></NER>
<NER>by <PER>John J. La Scala</PER>, <PER>Scott Bingham</PER>, <PER>Kevin S. Andrews</PER>, <PER>James M. Sands</PER>,</NER>
<NER>by <PER>John R. Mills</PER> 
 Strategic Insights is a bi - monthly electronic journal produced by the Center for Contemporary 
 Conflict at the Naval Postgraduate School in Monterey, California. The views expressed here are 
 those of the author (s) and do not necessarily represent the views of NPS, the Department of 
 Defense, or the U. S. Government. 
 For a PDF version of this article, click here</NER>
<NER>by <PER>John R. Mills</PER></NER>
<NER>by <PER>Justin Shumaker</PER></NER>
<NER>by <PER>Kristan Gurton</PER>, <PER>Melvin Felton</PER>, and <PER>Richard Tober</PER></NER>
<NER>by <PER>LTC Steven Creighton</PER>, Ph.D, <PER>Dr. Peter W. Chung</PER>, 
 and <PER>Dr. John D. Clayton</PER></NER>
<NER>by <PER>LTC Steven Creighton</PER>, Ph.D, <PER>Dr. Peter W. Chung</PER>,</NER>
<NER>by <PER>Lam Nguyen</PER></NER>
<NER>by <PER>M. W. Cole</PER>, <PER>E. Ngo</PER>, and <PER>R. J. Tan</PER></NER>
<NER>by <PER>MAJ. Petra McGregor</PER>, USAF 
 Strategic Insights is a monthly electronic journal produced by the Center for Contemporary 
 Conflict at the Naval Postgraduate School in Monterey, California. The views expressed here are 
 those of the author (s) and do not necessarily represent the views of NPS, the Department of 
 Defense, or the U. S. Government. 
 For a PDF version of this article, click here</NER>
<NER>by <PER>MAJ. Petra McGregor</PER>,</NER>
<NER>by <PER>Michael J. McQuaid</PER></NER>
<NER>by <PER>Nicholas Fung</PER> and <PER>Philip David</PER></NER>
<NER>by <PER>Pamela Bailey</PER></NER>
<NER>by <PER>Peter Munson</PER> [ 1</NER>
<NER>by <PER>Raymond E. Brennan</PER> and <PER>James M. Sands</PER></NER>
<NER>by <PER>Robert Looney</PER> 
 Strategic Insights is a monthly electronic journal produced by the Center for Contemporary 
 Conflict at the Naval Postgraduate School in Monterey, California. The views expressed here are 
 those of the author (s) and do not necessarily represent the views of NPS, the Department of 
 Defense, or the U. S. Government. 
 For a PDF version of this article, click here</NER>
<NER>by <PER>Robert Looney</PER></NER>
<NER>by <PER>Sam S. Chang</PER>, <PER>Dennis M. Garvey</PER>, 
 <PER>Chatt C. Williamson</PER>, and <PER>Giap Huynh</PER></NER>
<NER>by <PER>Sam S. Chang</PER>, <PER>Dennis M. Garvey</PER>,</NER>
<NER>by <PER>Stephen Blank</PER></NER>
<NER>by <PER>Tyrone Jones</PER> and <PER>Katsuyoshi Kondoh</PER></NER>
<NER>by Engdahl et al. Furthermore, the residuals resulting with our 3 ? D tables will comprise the primary data set</NER>
<NER>by Executive Order 13 22 6. The OSTP Director also plays a role in the communication</NER>
<NER>by LEAD</NER>
<NER>by Linking Strategic</NER>
<NER>by Novel Fabrication</NER>
<NER>by Socio-Demographic</NER>
<NER>by Thrombelastography Is Indicative of</NER>
<NER>by age and gender, active duty, US Army, 20 02</NER>
<NER>by amending OSTP ?s authorization statute. When Congress evaluates the various policy options</NER>
<NER>by at least 2 % annually through FY 20 15, increase non-petroleum</NER>
<NER>by maneuver. That type of combat is still largely</NER>
<NER>by spall above the explosion. A recent model by Fisk (20 06) shows the explosion S -wave spectra can</NER>
<NER>by terrorist attacks. (These techniques could also help</NER>
<NER>by the Air Force. They also earn their</NER>
<NER>by the Energy Security Task Force in September 20 06, where practical.</NER>
<NER>by the ensemble of clocks is called a t, ime scale. The question arises how to combine the</NER>
<NER>by the lack of these joint attributes.</NER>
<NER>by</NER>
<NER>c 10 0</NER>
<NER>c i</NER>
<NER>c r 1</NER>
<NER>c. This statement may not be used on technical documents that formerly were classified unless such documents are cleared</NER>
<NER>c</NER>
<NER>cabinet resignations, political assassinations, labor strikes, a war between Hezbollah and Israel,</NER>
<NER>cal exigencies cannot be ignored. And so we find the Army of</NER>
<NER>calculates the number and positioning of the depth banks</NER>
<NER>calculations by the buoy?s data processor permit reporting</NER>
<NER>calibrating the International Monitoring System (IMS) for the Comprehensive Nuclear-Test-Ban</NER>
<NER>called micro-Doppler [ 1 ]. Sources of micro-motions may be</NER>
<NER>cally ; and that ability, especially</NER>
<NER>cally tough, combat-oriented image of the</NER>
<NER>cally, there must be</NER>
<NER>camp at Kitali</NER>
<NER>campaign plan. Air operations (which</NER>
<NER>can be processed for 25 mm resolution at all ranges out to</NER>
<NER>can be set after the demands in the ?rst period are</NER>
<NER>can be tuned, improved long term performance could be</NER>
<NER>can delaminate and extrude after approximately 20 00</NER>
<NER>can make a difference</NER>
<NER>can provide this direction reference, NDBC has found</NER>
<NER>can support the increased fuel costs. Like the nation, DoD must focus on reducing</NER>
<NER>cancer (4). PRL binds its cognate single-membrane receptor, the PRL receptor (PRLR)</NER>