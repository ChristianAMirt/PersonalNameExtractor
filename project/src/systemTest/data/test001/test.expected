Character limit exceeded (4000). Try again with a shorter text